kolibri/plugins/device/assets/src/routes/index.js

```diff
@@ -75,6 +77,42 @@ const routes = [
       store.dispatch('preparePage', { name, isAsync: false });
     },
   },
+  {
+    name: PageNames.MANAGE_SYNC_SCHEDULE,
+    component: withAuthMessage(ManageSyncSchedule, 'superuser'),
+    path: '/facilities/:facilityId/managesync',
+    props: route => {
+      const facilityId = route.params.facilityId || store.getters.currentFacilityId;
+      return {
+        goBackRoute: { name: PageNames.FACILITIES_PAGE },
+        facilityId,
+        editSyncRoute: function(deviceId) {
+          return {
+            name: PageNames.EDIT_SYNC_SCHEDULE,
+            params: {
+              device_id: deviceId,
+              facilityId: facilityId,
+            },
+          };
+        },
+      };
+    },
+  },
+  {
+    name: PageNames.EDIT_SYNC_SCHEDULE,
+    component: withAuthMessage(EditDeviceSyncSchedule, 'superuser'),
+    path: '/facilities/:device_id/:facilityId/editdevice',
+    props: route => {
+      return {
+        goBackRoute: { name: PageNames.MANAGE_SYNC_SCHEDULE },
+        facilityId: route.params.facilityId || store.getters.currentFacilityId,
```

Comment:
This does not need to be changed, but just a comment that as the facilityId is a required parameter, I am not sure the `||` is needed here.

Reply:
I agree. cc @AllanOXDi